### ISSUE TITLE
Does LifterLMS incorrectly use the `[video]` shortcode to power video embeds in courses and lessons?

### ISSUE THREAD

**Issue by thomasplevy** [gocodebox/lifterlms#833]
I've noticed themes (like Astra) are filtering the HTML return of the `[embed]` shortcode to make video embeds automatically responsive.

They are not filtering `[video]` embeds (should they, I'm not sure, I need to do more research).

However, in digging into this to determine why LifterLMS videos in the course & lesson fields aren't automatically responsive on Astra I found this in the WordPress core: 

> * This implements the functionality of the Video Shortcode for displaying WordPress mp4s in a post.

[Source](https://core.trac.wordpress.org/browser/tags/5.1.1/src/wp-includes/media.php#L2499)

So I'm now wondering if we should be using `[embed]` as opposed to `[video]` and `[audio]` in our `get_embed()` methods on `LLMS_Post_Model` classes. [Source](https://github.com/gocodebox/lifterlms/blob/master/includes/abstracts/abstract.llms.post.model.php#L605)

<details>
<summary>Partial System Report</summary>

```
Theme
-------------------------------------------
Name: Astra
Version: 1.8.1
Themeuri: https://wpastra.com/
Authoruri: https://wpastra.com/about/
Template: 
Child Theme: No
Llms Support: Yes
Plugins
-------------------------------------------
Akeeba Backup for WordPress: 3.4.2.2
Astra Pro: 1.8.1
Astra Widgets: 1.1.0
Custom Fonts: 1.0.7
Import / Export Customizer Settings: 1.0.0
Jetpack by WordPress.com: 7.2.1
LifterLMS: 3.30.2
LifterLMS Helper: 3.0.2
LifterLMS WooCommerce: 2.0.9
Loco Translate: 2.2.2
LoginPress - Customizing the WordPress Login: 1.1.21
Mailchimp for WooCommerce: 2.1.15
Mollie Payments for WooCommerce: 5.1.6
Really Simple SSL: 3.1.5
Ultimate Addons for Gutenberg: 1.12.5
Uncanny LearnDash Toolkit: 3.0.2
WooCommerce: 3.5.8
WooCommerce PDF Invoices & Packing Slips: 2.2.11
WooCommerce Services: 1.19.0
WP-Optimize: 2.3.3
WPForms Lite: 1.5.2.2
```

</details>

Problem site was using Vimeo videos

**Comment by thomasplevy** [comment]
@Nikschavan, @sujaypawar or anyone at Brainstorm if you'd be willing to weigh in with any thoughts or opinions on this. As I noticed this as a result of a support request from a user using Astra with LifterLMS I'm doing my testing and thinking against Astra. If anyone over there has any opinions on whether or not we're misusing the core shortcode I'd love to hear them. Thanks as always!

**Comment by thomasplevy** [comment]
@actual-saurabh If you can read this over and give me your 2cents I'd appreciate it also

**Comment by Nikschavan** [comment]
> LifterLMS videos in the course & lesson fields aren't automatically responsive

When I am testing this, At least, Youtube/Vimeo/wistia videos added in the featured video setting of a course or a lesson appear responsive and take up complete content width. ([screenshot](https://cl.ly/ab6516644e47))

Are you referring to any particular video format or videos added from some other location?

**Comment by thomasplevy** [comment]
@Nikschavan 

Thanks for weighing in.

The video in question was a Vimeo embed. I traced it back (albeit it quickly) to what I thought was this issue (using `[video]` not `[embed]` but perhaps theres something else going on on this particular site.

I'll setup a more specific test case to work further. In the end I don't think this is an Astra issue at all but I was just curious if y'all had any thoughts on the usage of one shortcode over the other and if there's a semantic issue here.

I've added the plugin and theme info from our system report to the issue and I'll setup a better test case in the future.

Thanks again!
